File: jukengkengparent/jukengkeng-sellergoods-interface/src/main/java/com/jukengkeng/sellergoods/service/TypeTemplateService.java
package com.jukengkeng.sellergoods.service;

import com.jukengkeng.pojo.TbTypeTemplate;
import entity.pageResult;
import entity.result;

import java.util.List;
import java.util.Map;

/**
 * 模板管理
 */
public interface TypeTemplateService {

    //分页查询
    pageResult findPageList(TbTypeTemplate tbTypeTemplate, Integer pageNum, Integer pageSize);

    //保存
    void saveTypeTemplate(TbTypeTemplate tbTypeTemplate);
}

File: jukengkengparent/jukengkeng-manager-web/src/main/webapp/js/myAction/service/typeTemplateService.js
app.service("typeTemplateService", function ($http) {
    this.queryList = function (pageNum, pageSize, searchEntity) {
        return $http.post("../typeTemplate/findPageList.do?pageNum=" + pageNum + "&pageSize=" + pageSize, searchEntity);
    }

    //保存商品类型模板
    this.saveTypeTemplate = function (entity) {
        return $http.post("../typeTemplate/saveTypeTemplate.do",entity);
    }
});